composer.json

```diff
@@ -7,26 +7,29 @@
         "drush/drush": "^10.3",
         "drupal/purge": "^3.0@beta",
         "drupal/purge_queuer_url": "^1.0@RC",
-        "drupal/redis": "^1.0@RC",
-        "drupal/section_purge": "2.3-rc2",
+        "drupal/section_purger": "dev-updates_to_compatible_with_drupal_9",
+        "drupal/redis": "^1.0",
         "drupal/smtp": "^1.0@RC",
         "drupal/queue_mail": "^1",
-        "drupal/jwt": "1.0-beta1",
+        "drupal/jwt": "^1.0",
         "drupal/config_split": "^1.4"
     },
-    "repositories": {
-        "drupal": {
+    "exclude": [
+        "drupal/section_purger"
+    ],
+    "repositories": [
+        {
             "type": "composer",
-            "url": "https://packages.drupal.org/8"
+            "url": "https://packages.drupal.org/8",
+            "exclude": [
+                "drupal/section_purger"
+            ]
+        },
+        {
+            "type": "git",
+            "url": "https://git.drupalcode.org/issue/section_purge-3227034.git"
```

Comment:
@vincent-gao do we need this here, as the repo is going to get added to the root composer anyway using the ansible. Also the part exclude above, don't think we need this to add it in here. Checkout this test branch I tried and it works fine with the content-vic D9 test branch - https://github.com/dpc-sdp/baywatch/pull/44/files